fuzzing/fuzz/Cargo.toml

```diff
@@ -8,6 +8,10 @@ publish = false
 [package.metadata]
 cargo-fuzz = true
 
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--generate-link-to-definition", "--cfg=docsrs"]
```

Comment:
I love seeing my nightly features being used more and more. :)

Reply:
Thank you for implementing them! :)